routes/confirmation/checkAnswersFormat.js

```diff
@@ -0,0 +1,65 @@
+const { answerInfo } = require('./checkAnswers')
+const { hasData } = require('./../../utils')
+const { format, parseISO } = require('date-fns')
+const { routes } = require('./../../config/routes.config')
+
+const formatDataLine = (data, session) => {
+  if (data.length > 1) {
+    const dataValues = []
+    data.map((val) => {
+      dataValues.push(hasData(session, val, true))
+    })
+
+    if (typeof dataValues[1] === 'number') {
+      return dataValues.reduce((a,b) => a + b, 0)
+    } else if (typeof dataValues[1] === 'string') {
+      return dataValues.join(' ')
+    }
```

Comment:
Is it possible to not `return` here?

Ie, if `data.length > 1` is true, do we always have a number or a string?

Reply:
so far we only have it for retrieving name (firstName, lastName), and then for addingNumbers.

I guess the better assumption would be splitting out into functions to handle that return val